### DISCUSSION TITLE
Unable to submit application, page refreshing

### DISCUSSION THREAD

**Discussion by YourJailDev** [community/community#83942] [GitHub Education] [answered]
Hello, I'm not able to submit my GitHub student application. After I click "Process my Application" it just refreshes the page without submitting the application. Any ideas what going on?

**Comment by faridnec** [comment, marked as answer]
Can you provide more context or did any message appear after you refresh the page? Once I also had the same issue but then I solved it by taking a picture of my academic ID proof and then it worked. Had some issues when trying to submit using an uploaded document from the local files

**Comment by Boneheinwai6801** [comment]
Do this Again

**Comment by YourJailDev** [comment]
It's fine now. When I  tried submitting using my laptop.

**Comment by samus-aran** [comment]
I'll be closing this discussion as it appears you have found your answer. Please feel free to re-open if you continue to have this issue.

**Comment by javawolfpack** [comment]
I have tried three browsers now, I click process my application and it just keeps loading the submit application materials page. Seemingly no way to get support/report this beyond forums/here...
